src/switch_ivr.c

```diff
@@ -1071,6 +1072,7 @@ SWITCH_DECLARE(switch_status_t) switch_ivr_park(switch_core_session_t *session,
 			if ((var = switch_channel_get_variable(channel, SWITCH_SEND_SILENCE_WHEN_IDLE_VARIABLE))) {
 				sval = atoi(var);
 			}
+			switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_WARNING, "Reupdate park silence noise to %d\n", sval);
```

Comment:
perhaps only 'update park silence..' ?

Reply:
Its now possible to reset the silence value in park.
```
 <action application="set" data="send_silence_when_idle=100"/>
 ...
 <action application="park"/>
```
The above will set the initial comfort noise to 100 for park. But if somehow I made an application or execute a command that may change the value e.g. eavesdrop/three_way
```
  uuid_setvar <uuid> send_silence_when_idle=-1
  originate {...}[...]rtp &three_way(<uuid>);
```
The park command will pickup the new `send_silence_when_idle` value. So it can be just a Update or possible Reupdate of silence value. For simplicity, I just specify `Reupdate` in the log because if reach that particular code it may update or initiate silence value for Park.